remote.c

```diff
@@ -2226,6 +2370,14 @@ execute_cmd(struct daemon_remote* rc, RES* ssl, char* cmd, struct rc_state* rs)
 		do_assoc_tsig(ssl, rc->xfrd, skipwhite(p+10));
 	} else if(cmdcmp(p, "del_tsig", 8)) {
 		do_del_tsig(ssl, rc->xfrd, skipwhite(p+8));
+	} else if(cmdcmp(p, "add_cookie_secret", 17)) {
+		do_add_cookie_secret(ssl, rc->xfrd, skipwhite(p+18));
```

Comment:
```suggestion
		do_add_cookie_secret(ssl, rc->xfrd, skipwhite(p+17));
```
The string length is 17.